starter/slack-spring-boot/src/main/kotlin/com/kreait/slack/broker/store/user/UserJoinedEventReceiver.kt

```diff
@@ -0,0 +1,25 @@
+package com.kreait.slack.broker.store.user
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.kreait.slack.api.contract.jackson.SlackEvent
+import com.kreait.slack.api.contract.jackson.common.types.Member
+import com.kreait.slack.broker.receiver.EventReceiver
+import com.kreait.slack.broker.store.team.Team
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+
+/**
+ * EventReceiver that adds a user to the registered store when a new user joins the team
+ */
+class UserJoinedEventReceiver @Autowired constructor(private val userStore: UserStore) : EventReceiver {
+
+    override fun supportsEvent(slackEvent: SlackEvent): Boolean {
+        return slackEvent.event["type"] == "team_join"
```

Comment:
can you create an issue in RC1 to create a constant class that contains all Events?

Reply:
you already requested that once, and I already did that :D